webapp/portlet/src/main/webapp/vue-apps/search/components/SearchTagList.vue

```diff
@@ -37,7 +37,10 @@
             :value="tag"
             @click="handleTag(tag)"
             @keyup.enter="handleTag(tag)">
-            <span :class="`${isMobile ? 'primary--text' : ''}`"> {{ tag }}</span>
+            <span
+              :class="`${isMobile ? 'primary--text' : ''}`"
+              v-sanitized-html="tag">
```

Comment:
The tag shouldn't contain any html character. so the fix should be in tag creation process

Reply:
PR updated